Add a GameCube Backup Manager page. (#327)
RiiConnect24/Wii-Guide: _pages/en_US/gcbackupmanager.md

---
title: "GameCube Backup Manager"
---

{% include toc title="Table of Contents" %}

If you need help for anything regarding this tutorial, please join [the RiiConnect24 Discord server](https://discord.gg/rc24) (recommended) or [e-mail us at support@riiconnect24.net](mailto:support@riiconnect24.net).
{: .notice--info}

GameCube Backup Manager is a convenient PC software that simplifies the process of transferring GameCube games to your USB drive or SD Card. It functions similar to the popular tool called [Wii Backup Manager](wiibackupmanager), allowing you to easily manage and organize your GameCube game backups. With this program, you can effortlessly copy your favorite GameCube games onto portable storage devices, making them readily accessible for future gaming sessions.

#### What you need

- An SD card or USB drive
- [GameCube Backup Manager](https://github.com/AxionDrak/GameCube-Backup-Manager/releases)

To play multiple games, it is recommended to use an external hard drive for your Wii. Any external hard drive found on most modern marketplaces should work with your Wii.
{: .notice--info}

> Make sure your USB drive is formatted as FAT32. Do not format it as other types such as NTFS, exFAT, extFS, or WBFS, the latter being an outdated filesystem for storing Wii games.

Reviewer: ```suggestion
Make sure your USB drive is formatted as FAT32. Do not use other filesystems such as NTFS, exFAT, extFS, or WBFS.
```
"types" is a bit vague, so I would change that
Not sure it needs to be explained what WBFS is, but I think this is copied from another page so not your fault
I don't insist on removing that specifically
